c2tc-mobile/components/Tab.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+import { Text } from "react-native";
+
+import Tabs from "react-native-tabs";
+
+export default class Tab extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { page: "filter" };
+  }
+
+  getState() {
```

Comment:
what exactly are you returning? change your function name to be more descriptive.